compiler/src/dotty/tools/dotc/tastyreflect/CoreImpl.scala

```diff
@@ -73,8 +73,6 @@ trait CoreImpl extends scala.tasty.reflect.Core {
       type Refined = tpd.RefinedTypeTree
       type Applied = tpd.AppliedTypeTree
       type Annotated = tpd.Annotated
-      type And = tpd.AndTypeTree
```

Comment:
Should also be deleted from `scala/tasty/reflect/Core.scala`, its docs and `docs/docs/reference/other-new-features/tasty-reflect.md`.

Reply:
Still open

Reply:
Updated